src/unix/linux_like/linux/mod.rs

```diff
@@ -2759,6 +2759,11 @@ extern "C" {
         sevp: *mut ::sigevent,
     ) -> ::c_int;
 
+    pub fn clock_getcpuclockid(
+        pid: ::pid_t,
+        clk_id: *mut ::clockid_t
```

Comment:
Add a comma for style.

```suggestion
        clk_id: *mut ::clockid_t,
```